app/models/organisations_model.php

```diff
@@ -77,4 +77,22 @@ function get_organisations() {
         return $org_list;
     }
 
+    function delete_organisations($org_id) {
+        $org_delete = false;
+
+        $delete_result = $this->execute(
+            'DELETE FROM `organisations` WHERE id=?',
+            'i',
+            array(
+                &$org_id
+            )
+        );
+
+        if ($this->DB->affected_rows === 1) {
```

Comment:
Check `$delete_result` also,

maybe,

    if ($delete_result && $this->DB->affected_rows === 1)